### DISCUSSION TITLE
vercara.com: remnant hole above header, left behind by blocked annoyance

### DISCUSSION THREAD

**Discussion by DimitrisKalaboukas** [uBlockOrigin/uAssets#25441] [Q&A] [answered]
Could you, please, provide me with a custom rule to move header to the top? I don’t know if it should be added to a uBO list.

**Comment by gorhill** [comment, marked as answer]
vercara.com##.header-2023:style(margin-top:0!important)

It's not going to be added to uBO's list, this is a very minor subjective cosmetic quirk for a site unlikely to be of interest for the majority. Best is to just give feedback to the webmaster of the site.
